app/actions/GalleryPictureActions.ts

```diff
@@ -32,28 +32,6 @@ export const fetchGalleryPictures = (
     propagateError: true,
   });
 
-export function fetchSiblingGallerPicture(
-  galleryId: EntityId,
-  currentPictureId: EntityId,
-  next: boolean,
-) {
-  const rawCursor = `p=${currentPictureId}&r=${next ? 0 : 1}`;
-  const cursor = Buffer.from(rawCursor).toString('base64');
-  return callAPI({
-    types: GalleryPicture.FETCH_SIBLING,
-    endpoint: `/galleries/${galleryId}/pictures/`,
-    query: {
-      page_size: 1,
-      cursor,
-    },
-    schema: [galleryPictureSchema],
-    meta: {
-      errorMessage: 'Henting av bilde feilet',
-    },
-    propagateError: true,
-  });
-}
```

Comment:
Is this not useful anymore? I suppose it's nice to fetch the siblings when the current image is finished loading?

Reply:
Nah, it doesn't really make any sense as this just fetches one image at a time. We can just fetch a whole page of images at a time, like in the album-page. This just causes unneccessary requests.